apply_DDT.py

```diff
@@ -151,6 +175,7 @@ def main():
     #--------------------------------------------------------------------------
     # Grab the bkg data
     features_common = ["pt", "mt", "rho", "rt"]
+    #features_common = ["pt", "mt", "rho", "mass", "rt"]
```

Comment:
Looks like some commented out code creeped in, probably my fault, could you delete it?